### ISSUE TITLE
doesnot parse multiple events from ics file

### ISSUE THREAD

**Issue by abodhade** [rabidaudio/android-ical-parser#1]
I am facing issue when ics file contains multiple events it does not add to calender.

**Comment by rabidaudio** [comment]
Yeah the README notes this limitation. You can probably fix it by reading multiple VEvent props from the file. I'm not sure if there's a way to add multiple events to the intent though. 
Anyway I'm not sure this project is necessary anymore; I think the issue is fixed in recent versions of Android.

**Comment by abodhade** [comment]
could you please give link on which this issue is fixed for recent versions of Android.
